Dockerfile

```
@@ -1,15 +1,17 @@
 FROM python:3.11
 
-EXPOSE 8000
-
 WORKDIR /app
 
 COPY Pipfile /app
 COPY Pipfile.lock /app
 
-RUN ["pip", "install", "pipenv"]
-RUN ["pipenv", "install", "--dev", "--deploy", "--system"]
+RUN pip install pipenv
+RUN pipenv sync --dev --system
 
 COPY . /app
 
-CMD ["/app/docker_entrypoint.sh"]
+RUN python manage.py restoredb
+
+EXPOSE 8000
+
+ENTRYPOINT [ "daphne", "-b", "0.0.0.0", "-p", "8000", "webstrom.asgi:application" ]
```

toto by neslo tiez tym (ne)formatom?
```suggestion
ENTRYPOINT daphne -b 0.0.0.0 -p 8000 webstrom.asgi:application
```

---

Šlo by to, ale potom by sa to púšťalo vnútri shellu, a tam je napríklad problém s preposielaním signálov, takže ten container napríklad nejde veľmi rozumne vypnúť